pubtools/_ami/tasks/push.py

```diff
@@ -228,7 +228,7 @@ def upload(self, push_item):
             "virt_type": push_item.virtualization,
             "root_device_name": push_item.root_device,
             "volume_type": push_item.volume,
-            "billing_products": push_item.billing_codes.codes,
+            "billing_products": push_item.billing_codes.codes or None,
```

Comment:
I have problems with https://github.com/release-engineering/pushsource/pull/231 being a backwards-incompatible change.

If the idea there was to guarantee that push_item.billing_codes is non-None so it's safe to always check push_item.billing_codes.codes: I'd say it's not a strong enough reason to pursue a backwards-incompatible change.  They often have some nasty fallout you can't predict.  So even though the code here will look clumsy, I would suggest just fixing it here like:

```suggestion
            "billing_products": (push_item.billing_codes.codes or None) if push_item.billing_codes else None,
```